backtrace-library/src/main/java/backtraceio/library/watchdog/BacktraceWatchdog.java

```diff
@@ -75,7 +76,7 @@ public boolean checkIsAnyThreadIsBlocked() {
             }
 
             BacktraceLogger.w(LOG_TAG, String.format("Thread %s %s  might be hung, timestamp: %s",
-                    Long.toString(currentThread.getId()), currentThread.getName(), now_str));
+                    currentThread.getId(), currentThread.getName(), now_str));
```

Comment:
Android Studio gives a warning that `Long.toString` is unnecessary, IDE shows it in grey/disabled font, and formatting removes that method. I guess it will automatically cast it to `String`. Should we still change this @konraddysput ?

Reply:
I think it makes sense for warning about unnecessary Long.toString - however in the string.format method, shouldn't we now use %d instead of %s for this long?

Reply:
Will make this change, thank you!